src/svelte/src/components/ServerMetrics/ServerMetrics.svelte

```diff
@@ -20,13 +20,19 @@ limitations under the License.
 
   let heartbeat = {
     latency: 0,
-    omegaEditPort: 0,
     serverCpuLoadAverage: 0,
     serverTimestamp: 0,
     serverUptime: 0,
     serverUsedMemory: 0,
-    serverVersion: 'Unknown',
     sessionCount: 0,
+    omegaEditPort: 0,
+    serverVersion: 'Unknown',
```

Comment:
these `'Unknown'` values are different than the `'unknown'` values in `serverInfo`. Should they be the same?

Reply:
Nope.  One is info and the other is setup for displaying in the UI.  The difference in case can also be used to help troubleshoot what side a problem has occurred on (extension side vs. webview side).

Reply:
serverVersion here could be typed as say "Unknown" | `v${number}.${number}.${number}`.  I say we look more broadly to see where adding these kinds of types would be beneficial as a different issue.